Signal/src/AppDelegate.m

```diff
@@ -333,10 +334,39 @@ - (void)application:(UIApplication *)application
 - (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull NSUserActivity *)userActivity restorationHandler:(nonnull void (^)(NSArray * _Nullable))restorationHandler
 {
     if ([userActivity.activityType isEqualToString:@"INStartVideoCallIntent"]) {
+        DDLogInfo(@"%@ got start video call intent", self.tag);
```

Comment:
You know, when I added this `INStartVideoCallIntent` logic, I was trying to handle the case where the user taps the "video" button in the CallKit UI.  I find it surprising that If the user taps the "signal" button in the contacts app (which has a "video" icon), the Signal App receives a `INStartAudioCallIntent` activity.